### DISCUSSION TITLE
希望wiki能添加开发者说明

### DISCUSSION THREAD

**Discussion by qdu-eelab** [Richasy/Bili.Uwp#229] [问答]
使用VS2022，直接编译调试没有问题；
但使用“远程计算机”连接xbox，运行时VS报错“试图加载格式不正确的程序”；
尝试生成msixbundle，使用开发者模式安装到xbox，错误代码0x80070002；
查看msix内容，发现比pre-release里的附件小很多，尤其是缺少Richasy.dll；

希望wiki能添加开发者说明，描述正常的编译构建和发布流程，谢谢！
（后面想实现对xbox手柄操作的适配）

**Comment by Richasy** [comment]
尽管我一直想写文档，但由于只能在业余时间处理问题，且目前bug修复占据了我大部分精力。技术预览阶段暂不会添加wiki，预计国庆结束技术预览之后，会开始写

**Reply by Q-Tai-mu** [reply]
文档越全第三方的第三方更好维护
